youtube_dl/utils.py

```diff
@@ -337,34 +337,53 @@ def get_element_by_id(id, html):
 
 
 def get_element_by_class(class_name, html):
-    return get_element_by_attribute(
+    retval = get_elements_by_class(class_name, html)
+    if retval:
+        return retval[0]
+    else:
+        return None
+
+
+def get_element_by_attribute(attribute, value, html, escape_value=True):
+    retval = get_elements_by_attribute(attribute, value, html, escape_value)
+    if retval:
+        return retval[0]
+    else:
+        return None
+
+
+def get_elements_by_class(class_name, html):
+    return get_elements_by_attribute(
         'class', r'[^\'"]*\b%s\b[^\'"]*' % re.escape(class_name),
         html, escape_value=False)
 
 
-def get_element_by_attribute(attribute, value, html, escape_value=True):
+def get_elements_by_attribute(attribute, value, html, escape_value=True):
     """Return the content of the tag with the specified attribute in the passed HTML document"""
```

Comment:
"""Return the content of the first tag with the specified ID in the passed HTML document""" - okay?